.github/workflows/pr-validation-checks.yml

```diff
@@ -0,0 +1,46 @@
+name: PR Validation Checks
+
+on:
+  # pull_request:
+  #   types:
+  #     - opened
+  #     - edited
+  #     - synchronize
+  #   branches:
+  #     - release-next
+  #     - latest
```

Comment:
Any reason to keep this commented out code/config?

Reply:
useful when someone's developing against this workflow from a fork to test the workflow locally but other than that no.